src/ExpressionEvaluator/CSharp/Test/ResultProvider/TupleTests.cs

```diff
@@ -361,7 +340,6 @@ class C
 {
     (int A, int B) F = (1, 2);
     (int, int, int C) G = (1, 2, 3);
-    (int, int B, int, int D, int, int F, int, int H, int) H = (1, 2, 3, 4, 5, 6, 7, 8, 9);
```

Comment:
>(int, int B, int, int D, int, int F, int, int H, int) H = (1, 2, 3, 4, 5, 6, 7, 8, 9); [](start = 4, length = 86)

Please consider adding back this case.

Reply:
The EE won't proceed with errors. Do you suggest just adding a working 8-tuple for this case?

Reply:
Yes please.

---
In reply to: [330799663](https://github.com/dotnet/roslyn/pull/38929#discussion_r330799663) [](ancestors = 330799663)